src/client/components/DetailsDisplay.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from "react";
+import handleErrors from "../App";
+import { useParams } from "react-router-dom";
+
+export default function DetailsDisplay() {
+  const [meal, setMeal] = useState([]);
+  const [availableMeal, setAvailableMeal] = useState([]);
+  let [reservationForm, setReservationForm] = useState("");
+  const [error, setError] = useState("");
+  const [saveState, setSaveState] = useState(false);
+  const [message, setMessage] = useState("");
+  const today = new Date();
+  const date = `${today.getFullYear()}-${
+    today.getMonth() + 1
+  }-${today.getDate()}`;
+  let { id } = useParams();
+
+  let handleChange = (e) => {
+    let { name, value } = e.target;
+
+    setReservationForm({
+      ...reservationForm,
+      [name]: value,
+      meal_id: meal.id,
+      created_date: date,
+    });
+  };
+
+  let save = (e) => {
```

Comment:
All `let` variable declarations should be changed to `const` since they are not reassigned. 🙂